### PR TITLE
[new] add timecircles@1.5.3 with git auto-update, colse #5586

### PR DESCRIPTION
**Pull request by Amomo**
Hi @LeaYeh For #5586, I add a new lib with git auto-update config.
repo: https://github.com/wimbarelds/TimeCircles

From [General use](https://github.com/wimbarelds/TimeCircles#general-use) in README.md
We only need "TimeCircles.css" and "TimeCircles.js".
I also add their minified version in my commit.

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `24`, Star `180`, Fork `64`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

Would you mind checking this PR for me? Thank you~ :-D

### CODE REVIEW

File: ajax/libs/timecircles/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wimbarelds/TimeCircles"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wimbarelds/TimeCircles.git",
+    "basePath": "inc/",
+    "files": [
+      "TimeCircles*.+(css|js)"
```
Comment (PeterDaveHello) [inline ajax/libs/timecircles/package.json]: Why not `TimeCircles*` here?

Comment (Amomo) [inline ajax/libs/timecircles/package.json]: Is it fine not to point a specified file extension here?

Comment (PeterDaveHello) [inline ajax/libs/timecircles/package.json]: current ext looks okay, once you add prefix in filemap, and currently all the files are needed, I think it's fine to ignore ext for now.

Comment (Amomo) [inline ajax/libs/timecircles/package.json]: ok. :+1:

**Comment by Amomo** [comment]
@LeaYeh I have rebased it, could you check again? Thank you~

**Comment by LeaYeh** [comment]
i think this pr is ok, please rebase it again.

**Comment by Amomo** [comment]
@LeaYeh my branch is match to cdnjs's latest master branch, I think I don't need to rebase again...

**Comment by LeaYeh** [comment]
@PeterDaveHello could check this pr?

**Pull request merged**